builder/ecs/step_run_source_server.go

```diff
@@ -197,6 +189,13 @@ func (s *StepRunSourceServer) Cleanup(state multistep.StateBag) {
 	}
 
 	serverID := s.serverID
+	ui.Say(fmt.Sprintf("Detacheing the volume..."))
```

Comment:
please move this line into `detachServerVolume`

Reply:
done